azure/basic-aks-cluster/README.md

```diff
@@ -10,25 +12,42 @@ If you're new to XebiaLabs blueprints, check out:
 * [Get started with blueprints](https://docs.xebialabs.com/xl-platform/concept/get-started-with-blueprints.html)
 * [Get started with XL JetPack](https://docs.xebialabs.com/xl-platform/concept/get-started-with-xl-jetpack.html)
 
-## Prerequisites
+## Usage
+
+To use this blueprint, run `xl blueprint` in an empty directory and select:
+
+```plain
+azure/basic-aks
+```
+
+## Tools and technologies
+<!--Dev create this section for consistency with other blueprint readmes (see https://github.com/xebialabs/blueprints/tree/master/gcp/microservice-ecommerce for example)-->
+
+## Minimum required versions
 
-### Authenticate to Azure
+This blueprint version requires at least the below versions of the specified tools to work properly.
+
+- XL Deploy: Version 8.6.1
+- XL CLI: Version 8.6
 
-> Note: This blueprint requires your Azure login credentials in order to create the cluster.
+## Prerequisites
```

Comment:
I think Markdown best practice is to insert a blank line between headings and body text, and a blank line between a line of body text and a bulleted list. GitHub's own Markdown renderer might handle it without the blank line, but others won't, so it's just a way to make the Markdown more copy/pastable.

Reply:
agree

Reply:
found other instances and spaced them